examples/kubernetes/sharded-vtworker.sh

```diff
@@ -24,7 +24,7 @@ cat vtworker-pod-template.yaml | sed -e "$sed_script" | $KUBECTL create -f -
 set +e
 
 # Wait for vtworker pod to show up.
-until [ $($KUBECTL get pod -o template -t '{{.status.phase}}' vtworker 2> /dev/null) = "Running" ]; do
+until [ $($KUBECTL get pod -o template --template'{{.status.phase}}' vtworker 2> /dev/null) = "Running" ]; do
```

Comment:
My bad. It isn't even a whitespace change for which I can blame my editor. Adding them back.